examples/language-modeling/run_language_modeling.py

```diff
@@ -207,8 +209,8 @@ def main():
 
     if config.model_type in ["bert", "roberta", "distilbert", "camembert"] and not data_args.mlm:
         raise ValueError(
-            "BERT and RoBERTa-like models do not have LM heads but masked LM heads. They must be run using the --mlm "
-            "flag (masked language modeling)."
+            "BERT, RoBERTa and XLNet-like models do not have LM heads but masked LM heads. They must be run using the"
```

Comment:
I believe you mean "BERT, XLNet and RoBERTa-like models" since `XLM``XLMRoBERTa` are so-called RoBERTa-like models.

Reply:
Yes, that's a typo, sorry